wpl_cli/README.md

```diff
@@ -0,0 +1,114 @@
+# Wen Program Library CLI
+
+A command line interface for WEN New Standard program.
+
+---
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Instructions](#instructions)
+
+---
+
+## Installation
+
+One can build directly from source using `cargo`. Prerequisites for the binary to work is installing Rust on one's machine.
+
+Install Rust via the command
+
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+---
+
+## Instructions
+
+The user is welcomed with the initial helper section via `wpl-cli` or `wpl-cli --help`.
+
+![image](https://github.com/wen-community/wen-program-library/assets/32637757/84922290-efed-462c-bb2f-b1c6f39472f4)
+
+## Collection commands
+
+The following commands help a user create/update and grouping assets within a collection.
+
+![image](https://github.com/wen-community/wen-program-library/assets/32637757/68d2fd54-40e3-4e73-8cf1-d367c433d837)
+
+### Get Collection
+
+![image](https://github.com/wen-community/wen-program-library/assets/32637757/f03269d0-baea-46cd-afec-17421d4baf14)
+
+### Create collection
+
+![image](https://github.com/wen-community/wen-program-library/assets/32637757/65e68b03-4510-419f-af71-07c1925b7bc7)
+
+### Update collection
+
+![image](https://github.com/wen-community/wen-program-library/assets/32637757/51130ae9-d053-4b42-b317-a71dec7a3ef9)
+
+### Asset grouping sub commands
+
+![image](https://github.com/wen-community/wen-program-library/assets/32637757/d6227d80-45b8-4882-a549-a8c078b069ad)
+
+### Add asset
+
+![image](https://github.com/wen-community/wen-program-library/assets/32637757/0824d5f4-8fee-441e-ace1-a473676acfa9)
+
+### Remove asset
+
+![image](https://github.com/wen-community/wen-program-library/assets/32637757/17cc9ff3-94e1-4908-a849-8947b619d200)
+
```

Comment:
Keeping a list of all available commands will tough to keep up to date. Instead just not that users can run --help to get info on required params and then use rust comments over each command declaration so prints out the description of the command.